src/Gestures/GesturePropertyTransition.vala

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright 2024 elementary, Inc. (https://elementary.io)
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ *
+ * Authored by: Leonhard Kargl <leo.kargl@proton.me>
+ */
+
+/**
+ * A class that will animate a property of a {@link Clutter.Actor} one to one with a gesture or
+ * with easing without a gesture. Respects the enable animation setting.
+ */
+public class Gala.GesturePropertyTransition : Object {
+    public delegate void DoneCallback ();
+
+    /**
+     * Emitted when all animations are finished that is when the property has reached the target value
+     * either via gesture or via easing or combined.
+     */
+    public signal void done ();
```

Comment:
As far as I can see this signal is used only inside the class, the users of the class seem to use `done_callback`. Should this signal be private instead?